pkg/gadgets/run/tracer/run.go

```diff
@@ -41,6 +44,7 @@ import (
 const (
 	ProgramContent  = "prog"
```

Comment:
I forgot to add `Param` as a prefix. So it would be `ParamProgramContent` instead